src/kv/store.h

```diff
@@ -241,14 +301,16 @@ namespace kv
       {
         std::lock_guard<SpinLock> mguard(maps_lock);
 
-        for (auto& map : maps)
+        for (auto& it : maps)
```

Comment:
`for (auto& [_, map] : maps)` should work I would have thought?

Reply:
No, the value here is a pair, so the `map` is actually `it.second.second`. Ideally we'd have nested structured bindings so I could write something like `auto& [_, [v, map]] : maps`, but this isn't supported yet so we're stuck with 2 lines.

Reply:
Ah right